src/lib.rs

```diff
@@ -119,7 +119,7 @@ pub async fn mirror(config: CondaMirrorConfig) -> miette::Result<()> {
     let subdirs = get_subdirs(&config, client.clone()).await?;
     tracing::info!("Mirroring the following subdirs: {:?}", subdirs);
 
-    let max_parallel = 32;
+    let max_parallel = config.number_of_threads_per_subdir as usize;
```

Comment:
`rattler_config` allows us to read the pixi global config which contains maximum concurrency: https://pixi.sh/latest/reference/pixi_configuration/#concurrency

we should read that one instead IMO

see also #97

do you want to do this? Otherwise i will take care of this eventually (hopefully in the coming weeks)

Reply:
So the reading of a pixi config should be optional given that not everybody would be using pixi?